handbook/engineering/search/product.md

```diff
@@ -38,6 +38,16 @@ We work in two-week iterations.
 
 Iterations start every other Monday.
 
+#### Planning artifacts
+
+We use two main artifacts to plan iterations: our [planning document](https://docs.google.com/document/d/1swnkQwd724IB_HP3_Mw3KkFnfl45t-PmWcNo_1gtkQM/edit#), and a GitHub project specific to the iteration. We think of their relative roles as follows:
+
+- The planning document is used as a discussion platform to agree on intended outcomes for the iteration. An intended outcome is not "issue #xxxx is closed", but rather "problem X is solved".
+- The GitHub project is an itemized representation of the work needed to accomplish the outcomes listed in the planning document.
+- If some items in the GitHub project are not closed by the end of the iteration, teammates should mention in their last weekly update of the iteration what this impact is on planned outcomes.
```

Comment:
Feels like there's a missing bullet on when and how we add items to the planning document. Most recently it was during the sync (Loic asks us), and the document says "EM drafts this". I actually feel pretty unclear about this still :-) For upcoming iterations, I'd personally like to jot down a couple of things that I'd like to motivate and balance (e.g., contribution time to search UX work, balanced with backend work that is crucial, follow up work for predicates where we have a big opportunity, etc.). This would be an outlook of work and time allocation for, say, 3 months forward. I'd be happy to chuck this in a document to help inform planning. Don't know if others feel this way. Anyway the main question here is about "how and when do we populate the planning document, and how can we approach populating it in a way that takes into account a longer prospective of work to be done?". Product board is reflective of longer term goals, but not technical tasks that bear on them, or previous product features that have follow up technical work or maintenance, and communicating/planning around those technical tasks are what's on my mind.

Reply:
https://media1.giphy.com/media/8rEFo5ZrPaeBolyIah/giphy.gif?cid=5a38a5a2nnw8kmzsnw14zis5hp7dc8cqtlig2ldr9y8s3mj4&rid=giphy.gif&ct=g

sorry for block of text 🥉

Reply:
@rvantonder the existing [handbook documentation](https://about.sourcegraph.com/handbook/engineering/search/product#planning) (kept here under "planning a new iteration") covers precisely that.

Reply:
OK, I suppose this section doesn't need to link to planning-a-new-iteration since it's about artifacts, so this makes sense. Feels like there's still a gap for:

> how can we approach populating it in a way that takes into account a longer prospective of work to be done?

I'm going to make my best guess how to approach this and outline a doc for technical work/time allocation for myself that goes beyond 2-weeks-at-a-time scope, and maybe that can integrate into the planning doc.